IBAnimatableApp.xcodeproj/project.pbxproj

```diff
@@ -920,7 +926,7 @@
 		AE6B015F1C2A082300950BB2 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				APPLICATION_EXTENSION_API_ONLY = YES;
+				APPLICATION_EXTENSION_API_ONLY = NO;
```

Comment:
Please revert that then.

Jake

Sent from my iPhone

> On 21 Jul 2016, at 8:41 AM, Jason Ak <notifications@github.com> wrote:
> 
> In IBAnimatableApp.xcodeproj/project.pbxproj:
> 
> > @@ -920,7 +926,7 @@
> >  		AE6B015F1C2A082300950BB2 /* Debug */ = {
> >  			isa = XCBuildConfiguration;
> >  			buildSettings = {
> > -				APPLICATION_EXTENSION_API_ONLY = YES;
> > +				APPLICATION_EXTENSION_API_ONLY = NO;
> I Don't understand why this has benn changed.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>